server/src/main/java/com/objectcomputing/checkins/services/member_skill/skillsreport/SkillsReportServicesImpl.java

```diff
@@ -108,7 +112,8 @@ public SkillsReportResponseDTO report(@NotNull SkillsReportRequestDTO request) {
                 final TeamMemberSkillDTO dto = new TeamMemberSkillDTO();
                 dto.setId(memberId);
 
-                final String memberName = memberProfileRepo.findNameById(memberId);
+                //final String memberName = memberProfileRepo.findNameById(memberId);
+                final String memberName = memberProfileServices.getName(memberId);
```

Comment:
getName is undefined in the code.

Reply:
Yeah, I'm updating the related classes. This isn't complete yet.